src/main/java/com/demo/MybatisApplication/model/TeacherEntity.java

```diff
@@ -14,4 +14,5 @@ public class TeacherEntity {
     private String teacherName;
```

Comment:
Please make sure that the review comments are addressed/followed in all places and not just the highlighted places.

Reply:
Done